src/main/resources/io/debezium/connector/jdbc/build.version

```diff
@@ -0,0 +1 @@
+version=1.9.0.-SNAPSHOT
```

Comment:
Value should be provided via resource filtering.

Reply:
Could you please explain more?

Reply:
There is a maven construct that allows you to define variables inside resource files and then during the build process, those variables get substituted with real values.  Take a look at the `debezium-quarkus-outbox` runtime artifact.  You'll see I have 2 resource folders, and one is suffixed with "-filtered".  Those files get processed through the resource filtering step and the parsed version information gets automatically replaced so that the final file in the output artifact has the version information.